docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -134,15 +163,12 @@ entryPoints:
   https:
     address: ":443"
 
-serversTransport:
-  insecureSkipVerify: true
-
 providers:
   docker:
     endpoint: "unix:///var/run/docker.sock"
     exposedByDefault: false
   file:
-    filename: /static.yml
+    filename: '/static.yml'
```

Comment:
```suggestion
    filename: '/dynamic.yml'
```

Let's name this accordingly so it's less ambiguous.

An alternative would be to use:

```yaml
file:
  directory: /etc/traefik/config
  watch: true
```

Which means users can create dynamic configuration in this location with `.yml` or `.yaml` suffixes and they will (un)load as created/updated/deleted.